buildSrc/src/main/kotlin/Libs.kt

```diff
@@ -42,6 +42,7 @@ object Libs {
     // Coroutines
     const val COROUTINES_CORE = "org.jetbrains.kotlinx:kotlinx-coroutines-core:${Versions.COROUTINES}"
     const val COROUTINES_ANDROID = "org.jetbrains.kotlinx:kotlinx-coroutines-android:${Versions.COROUTINES}"
+    const val COROUTINES_PLAY_SERVICE = "org.jetbrains.kotlinx:kotlinx-coroutines-play-services:${Versions.COROUTINES_PLAY_SERVICE}"
```

Comment:
Why do you need this ?

Reply:
I will need it in Firestore but for a time being , i don't need it. Removed it.